src/scriptgen.unit.test.ts

```diff
@@ -66,14 +75,21 @@ describe("command generation", () => {
         expect(
             actual.includes("'CoberturaCodeCoverage','code-coverage/coverage.xml'")
         ).toBeTruthy();
+        expect(
+            actual.includes("'HTMLCodeCoverage','code-coverage/HTMLcoverage.html'")
```

Comment:
`code-coverage/coverage.html` or `code-coverage/codecoverage.html` for consistency?